### PR TITLE
Models Resolved

### PR DESCRIPTION
**Pull request by Jash271**
Models created, kindly review it and let me know what you think of it. I have also created a superuser
username:admin
password: admin
Also Enabled CORS

### CODE REVIEW

File: salespersonTracker/urls.py [hunk 1]
```diff
@@ -14,8 +14,17 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
+from django.contrib import admin
+from django.urls import include, path
```
Comment (Rashmil-1999) [inline salespersonTracker/urls.py]: please remove duplicate imports

File: salespersonTrackerREST/models.py [hunk 2]
```diff
@@ -1,3 +1,109 @@
...
+    Description=models.TextField()
+
+    def __int__(self):
+        return self.Item_Code
+    
+
+
+class ItemAssign(models.Model):
+    Item_Ref=models.ForeignKey(Item,on_delete=models.CASCADE)
+    Assigned_By=models.ForeignKey(Manager,on_delete=models.CASCADE)#Doubt about cascade
```
Comment (Rashmil-1999) [inline salespersonTrackerREST/models.py]: Good to go with cascade as you will remove items assigned to the salesman if the manager is deleted.

File: salespersonTracker/urls.py (hunk 1 above)
Comment (Jash271) [reply salespersonTracker/urls.py]: Done

File: salespersonTrackerREST/models.py (hunk 2 above)
Comment (Jash271) [reply salespersonTrackerREST/models.py]: models.SET_NULL attribute added, where ever applicable

**Comment by Jash271** [comment]
Models have been resolved, redundancy in urls.py resolved..

**Pull request merged**